tests/greenheart/test_hydrogen/test_greenheart_system.py

```diff
@@ -95,175 +97,152 @@ def test_simulation_wind_wave(subtests):
     filename_greenheart_config_onshore = os.path.join(
         orbit_library_path, f"plant/greenheart_config_onshore.yaml"
     )
 
-    lcoe, lcoh, _, _ = run_simulation(
-        filename_hopp_config,
-        filename_greenheart_config_onshore,
-        filename_turbine_config,
-        filename_orbit_config,
-        filename_floris_config,
+    config = GreenHeartSimulationConfig(
+        filename_hopp_config=filename_hopp_config,
+        filename_greenheart_config=filename_greenheart_config_onshore,
+        filename_turbine_config=filename_turbine_config,
+        filename_orbit_config=filename_orbit_config,
+        filename_floris_config=filename_floris_config,
         verbose=False,
         show_plots=False,
         save_plots=False,
         use_profast=True,
         post_processing=False,
         incentive_option=1,
-        plant_design_scenario=1,
+        plant_design_scenario=9,
         output_level=5,
     )
+    lcoe, lcoh, _, _ = run_simulation(config)
 
+    # TODO base this test value on something
     with subtests.test("lcoh"):
-        assert lcoh == approx(
-            7.057994298481547
-        )  # TODO base this test value on something
+        assert lcoh == approx(3.4745099686186)  
 
+    # TODO base this test value on something
     with subtests.test("lcoe"):
-        assert lcoe == approx(
-            0.10816180445700445
-        )  # TODO base this test value on something
+        assert lcoe == approx(0.04555763524550727)
 
 
 def test_simulation_wind_onshore_steel_ammonia(subtests):
-    turbine_model = "osw_18MW"
-    filename_turbine_config = os.path.join(
-        orbit_library_path, f"turbines/{turbine_model}.yaml"
-    )
-    filename_orbit_config = os.path.join(
-        orbit_library_path, f"plant/orbit-config-{turbine_model}.yaml"
-    )
-    filename_floris_config = os.path.join(
-        orbit_library_path, f"floris/floris_input_{turbine_model}.yaml"
-    )
     filename_greenheart_config = os.path.join(
         orbit_library_path, f"plant/greenheart_config_onshore.yaml"
     )
-    filename_hopp_config = os.path.join(orbit_library_path, f"plant/hopp_config.yaml")
-
-    lcoe, lcoh, steel_finance, ammonia_finance = run_simulation(
-        filename_hopp_config,
-        filename_greenheart_config,
-        filename_turbine_config,
-        filename_orbit_config,
-        filename_floris_config,
+
+    config = GreenHeartSimulationConfig(
+        filename_hopp_config=filename_hopp_config,
+        filename_greenheart_config=filename_greenheart_config,
+        filename_turbine_config=filename_turbine_config,
+        filename_orbit_config=filename_orbit_config,
+        filename_floris_config=filename_floris_config,
         verbose=False,
         show_plots=False,
         save_plots=False,
         use_profast=True,
         post_processing=False,
         incentive_option=1,
-        plant_design_scenario=1,
+        plant_design_scenario=9,
```

Comment:
why the design scenario change? I'm assuming just to include ammonia in the scenario?

Reply:
I checked this and it is correct